python/packages/autogen-core/src/autogen_core/models/_model_client.py

```diff
@@ -38,10 +40,12 @@ class ModelFamily:
     UNKNOWN = "unknown"
 
     ANY: TypeAlias = Literal[
+        "gpt-41",
```

Comment:
Hi, @ekzhu .
You have to update `is_openai` function as well. Please refer my PR https://github.com/microsoft/autogen/pull/6315/files#diff-bd837e130485115db4cf3793e3b4b2bce04e5ad042caad882159b91f78b0ec18L85-L91.